pkg/service/test/test.go

```diff
@@ -143,13 +151,16 @@ func (t *tester) Test(path, testReportPath string, appCmd, appContainer, appNetw
 	// stop listening for the eBPF events
 	loadedHooks.Stop(true, nil)
 
+	//Call the telemetry events.
+	tele.Testrun(resultForTele[0], resultForTele[1])
```

Comment:
If user deletes few test-case, only few mocks will not be used. This will give wrong data and also as explained before this will give wrong data because you are using it in proxy layer.

Reply:
I think you are talking about mocktest right? Here we are just passing the number of tests passed and failed, nothing to do with the mocks.